ebos/ecltransmissibility_impl.hh

```diff
@@ -93,14 +93,16 @@ EclTransmissibility(const EclipseState& eclState,
                     const Grid& grid,
                     std::function<std::array<double,dimWorld>(int)> centroids,
                     bool enableEnergy,
-                    bool enableDiffusivity)
+                    bool enableDiffusivity,
+                    bool enableDispersivity)
       : eclState_(eclState)
       , gridView_(gridView)
       , cartMapper_(cartMapper)
       , grid_(grid)
       , centroids_(centroids)
       , enableEnergy_(enableEnergy)
       , enableDiffusivity_(enableDiffusivity)
+      , enableDispersivity_(enableDispersivity)
 {
     const UnitSystem& unitSystem = eclState_.getDeckUnitSystem();
     transmissibilityThreshold_  = unitSystem.parse("Transmissibility").getSIScaling() * 1e-6;
```

Comment:
Again if I think this arbitrary constant should be moved to a common place. Maybe mark with a comment which is easy to search for when one agree on how to do this.